### DISCUSSION TITLE
Use of Discussions vs Issues

### DISCUSSION THREAD

**Discussion by ryankopf** [loco-rs/loco#364] [General]
Most other projects don't use Github's discussions to plan features, because discussions offer less visibility than issues and often receive less participation.

I would suggest we keep important discussions relating to the roadmap of the framework under "Issues".

**Comment by jondot** [comment]
Thanks for your feedback. We start by saying that one can open issues and we then triage it into the various buckets, so when in doubt -- just open an issue.
Issues are addressed as actionable things, in which you're correct - they get more attention.

However we scan the issues and discussions frequently, when discussions become actionable they get converted to issues, and when issues become un-actionable they get converted to discussions until they're "hot enough" to take action on.

The best thing is to submit PR of course, it is actionable and tangible.

If you have an issue that was converted to a discussion but you need more reasoning feel free to ask, we may have made a bad judgement or we can attempt to explain.

**Comment by ryankopf** [comment]
This was not about one particular item, but more meant to be a "Discussions" discussion about which is better, particularly for core features of the project, and whether or not "Discussions" will work out.

For example, I started a discussion on starter templates 2 days ago, but there hasn't been much discussion there yet. I'm not in a hurry for answers or anything like that, but I just noticed because "Discussions" does not have open items and a count on the web interface, and people accustomed to "Issues", I worry important "Discussions" may go overlooked in the future if they are not "Issues".
